values.yaml

```diff
@@ -130,4 +130,9 @@ tolerations: []
 affinity: {}
 
 # -- Extra lables to be added to all resources
-extraLabels: {}
+extraLabels:
```

Comment:
I'd rather keep it empty. Default values should be as clean as possible imo, giving minimal working setup